wpilibc/athena/src/IterativeRobot.cpp

```diff
@@ -41,6 +41,8 @@ void IterativeRobot::StartCompetition() {
   while (true) {
     // Call the appropriate function depending upon the current robot mode
     if (IsDisabled()) {
+      // wait for driver station data so the loop doesn't hog the CPU
+      m_ds.WaitForData();
```

Comment:
I think this shouldn't be in the IsDisabled() section?  It's not in the Java changes.